routing/unified_policies.go

```diff
@@ -208,13 +226,17 @@ func (u *unifiedPolicy) getPolicyNetwork(
 
 	var (
 		bestPolicy  *unifiedPolicyEdge
-		maxFee      lnwire.MilliSatoshi
+		maxFee      int64 = math.MinInt64
 		maxTimelock uint16
 	)
 
 	for _, edge := range u.edges {
+		// Add inbound fee to the amount that is sent over the channel.
+		amtWithInboundFee := amt +
+			lnwire.MilliSatoshi(edge.inboundFees.CalcFee(amt))
```

Comment:
I think this is not correct, because the inbound fee needs to be calculated over the amt to forward 😬